Description: minor removal of unsafe shell script
 eval in these segements unnecessary
Author: Alastair McKinstry <mckinstry@debian.org>
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=866631
Last-Updated: 2020-01-22
Forwarded: no

Index: libtool/build-aux/funclib.sh
===================================================================
--- libtool.orig/build-aux/funclib.sh
+++ libtool/build-aux/funclib.sh
@@ -322,7 +322,7 @@ EXIT_SKIP=77	  # $? = 77 is used to indi
 # putting '$debug_cmd' at the start of all your functions, you can get
 # bash to show function call trace with:
 #
-#    debug_cmd='eval echo "${FUNCNAME[0]} $*" >&2' bash your-script-name
+#    debug_cmd='echo "${FUNCNAME[0]} $*" >&2' bash your-script-name
 debug_cmd=${debug_cmd-":"}
 exit_cmd=:
 
Index: libtool/build-aux/ltmain.sh
===================================================================
--- libtool.orig/build-aux/ltmain.sh
+++ libtool/build-aux/ltmain.sh
@@ -387,7 +387,7 @@ EXIT_SKIP=77	  # $? = 77 is used to indi
 # putting '$debug_cmd' at the start of all your functions, you can get
 # bash to show function call trace with:
 #
-#    debug_cmd='eval echo "${FUNCNAME[0]} $*" >&2' bash your-script-name
+#    debug_cmd='echo "${FUNCNAME[0]} $*" >&2' bash your-script-name
 debug_cmd=${debug_cmd-":"}
 exit_cmd=:
 
